libs/httpproxy/proxy-camel-route/pom.xml

```diff
@@ -77,6 +77,10 @@
             <artifactId>common-system</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.servicemix.bundles</groupId>
+            <artifactId>org.apache.servicemix.bundles.jsr305</artifactId>
+        </dependency>
```

Comment:
Remove dependency. Should not be necessary.

Reply:
Investigated. It seems that one of the camel dependencies seems to be mucking with how we normally inherit this dependency -- so leaving this in as is fixes the problem.